### PR TITLE
[may_minihttp] support db client thread id calc

### PR DESCRIPTION
**Pull request by Xudong-Huang**
1. db connection match thread local one
2. update rust to 1.67

### CODE REVIEW

File: frameworks/Rust/may-minihttp/Cargo.toml [hunk 1]
```diff
@@ -17,15 +17,18 @@ nanorand = { version = "0.7", default-features = false, features = ["std", "wyra
 buf-min = { version = "0.7", features = ["bytes"] }
 yarte = { version = "0.15", features = ["bytes-buf", "json"] }
 
-may = { version = "0.3.25", default-features = false }
+may = "0.3"
 may_postgres = { git = "https://github.com/Xudong-Huang/may_postgres.git" }
```
Comment (volyrique) [inline frameworks/Rust/may-minihttp/Cargo.toml:21]: This and the following line appear not to use a fixed version of the dependency - you should change them to refer to a specific commit ID.

File: frameworks/Rust/may-minihttp/Cargo.toml [hunk 2]
```diff
@@ -17,15 +17,18 @@ nanorand = { version = "0.7", default-features = false, features = ["std", "wyra
...
 codegen-units = 1
 panic = 'abort'
-lto = "thin"
+lto = 'thin'
 debug = false
 incremental = false
 overflow-checks = false
+
+[patch.crates-io]
+may = { git = "https://github.com/Xudong-Huang/may.git", branch = "no_default_features" }
```
Comment (volyrique) [inline frameworks/Rust/may-minihttp/Cargo.toml:34]: Ditto.

Comment (Xudong-Huang) [reply frameworks/Rust/may-minihttp/Cargo.toml]: already remove this, please see the latest code.

File: frameworks/Rust/may-minihttp/Cargo.toml (hunk 1 above)
Comment (Xudong-Huang) [reply frameworks/Rust/may-minihttp/Cargo.toml]: done

**Pull request merged**